docs/roles/check-role-permissions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Check a Role's Permissions
+intro: Check a role's permissions to understand the role's access level on the associated project.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
+1. Select a `Project`
+1. Find the `Roles` card
+1. Click `View` in the `Roles` card to see a list of existing `Roles`
+1. In the list of `Roles` find the `Role` you want to check the permissions and click `DETAILS`
+1. In the role details view, find the permissions list
```

Comment:
Perhaps we need some closing sentence?
I miss the same closing sentence on topic `users/find-your-activity-summary/`, as user may lose focus if we don't say clearly: "look here", "copy the response from this place", etc.

Options I see:

```
1. In the role details view, find the permissions list
2. Under the peermissions list, you will see the list of permissions granted by this role
```

```
1. In the role details view, you will see the list of permissions granted by this role
```

Reply:
I agree with your review and I like option 2